modules/afamqp/CMakeLists.txt

```diff
@@ -1,36 +1,24 @@
 module_switch(ENABLE_AFAMQP "Enable afamqp module" RabbitMQ_FOUND)
 if (ENABLE_AFAMQP)
 
-set(AFAMQP_HEADERS
+set(AFAMQP_SOURCES
```

Comment:
Please keep the headers and source file list separet as currently the only use for headers is to install them. (and if you use IDE generator)

Reply:
Yes, and this separation is also broken in some modules.

Reply:
Final conclusion here is to
 * merge private headers with sources (and this is better for IDEs)
 * use `X_HEADERS` for public headers (which we want to be installed with syslog-ng)